buildspec.yml

```diff
@@ -0,0 +1,21 @@
+version: 0.2
+env:
+  variables:
+    # Agree to send anonymized reporting data to lambdagems
+    JETS_AGREE: 'yes'
+phases:
+  install:
+    runtime-versions:
+      ruby: 2.6
+    commands:
+      # Jets doesn't yet support ruby 2.7
+      # Let's temporarily install 2.5 until we can upgrade
+      #   https://github.com/tongueroo/jets/issues/444
+      - rbenv install 2.5.8
```

Comment:
I see the comment mentions Ruby 2.5, but the runtime-versions variable says 2.6. Is that just a comment issue or do we need to change one of those?

Reply:
Eh. I made them different on purpose, but didn't really explain why.

There's an open issue for jets to support ruby 2.7. When it does, we will get a dependabot pr and make the upgrade. If I made the runtime version 2.7, then the build would just work and no one would look at this file and there's a good chance that we'll keep installing 2.5.8 for no reason. By making the runtime version 2.6, when jets gets updated the build will break and someone will have to look at this file. Hopefully they'll see this comment and stop installing 2.5 which will drastically speed up the build.

Not sure if that's worth it or it just leads to confusion, but that's why I made them different 🤷‍♂️

Reply:
Oh and as to why the runtime version isn't just 2.5, aws only supports 2.6 and 2.7 in this build container. That's why 2.5 is being manually installed.

Reply:
@nvengal Technology is great, but sometimes we have to bend it a little bit to get what we want! Thanks for the explanation, and that is fine logic. I'm also glad we had this add-on conversation because we'll know we aren't crazy when we run into that future situation.